arches/install/arches-templates/project_name/settings.py-tpl

```diff
@@ -282,6 +283,10 @@ EMAIL_HOST_USER = "xxxx@xxx.com"
 
 DEFAULT_FROM_EMAIL = EMAIL_HOST_USER
 
+EXTRA_EMAIL_CONTEXT = {"salutation": "Hi",
```

Comment:
formatting nit:

```
EXTRA_EMAIL_CONTEXT = {
    "salutation": "Hi",
    "expiration":(datetime.now() + timedelta(seconds=CELERY_SEARCH_EXPORT_EXPIRES)).strftime("%A, %d %B %Y")
}
```

Reply:
we should make sure that all text is localizable: eg. "salutation"